python/python/glide/async_commands/cluster_commands.py

```diff
@@ -514,3 +514,43 @@ async def flushall(
             TClusterResponse[TOK],
             await self._execute_command(RequestType.FlushAll, args, route),
         )
+
+    async def copy(
+        self,
+        source: str,
+        destination: str,
+        replace: Optional[bool] = None,
+    ) -> bool:
+        """
+        Copies the value stored at the `source` to the `destination` key. When `replace` is True,
+        removes the `destination` key first if it already exists, otherwise performs no action.
+
+        See https://valkey.io/commands/copy for more details.
+
+        Note:
+            When in cluster mode, both `source` and `destination` must map to the same hash slot.
```

Comment:
Maybe
```suggestion
            Both `source` and `destination` must map to the same hash slot.
```
Because this very function is in cluster client only